### ISSUE TITLE
Not able to unbind s in visual mode

### ISSUE THREAD

**Issue by pzzxqj** [tridactyl/tridactyl#4970]
After typing `:unbind --mode=visual s`,  `s` still exists in visual map but not appear `:viewconfig`.

[20240419_153601.webm](https://github.com/tridactyl/tridactyl/assets/107164025/41c12804-d285-4455-9336-c03b72c9f6b8)

version: 1.24.1

**Comment by GodSaveEarth** [comment]
Yes, I confirm that. Unbinding any key bind (visual mode) restores it to default command, instead of clear.

**Comment by GodSaveEarth** [comment]
**Seems working now.** I moved to new Firefox installation and updated Tridactyl to 1.24.2 and the problem has gone.

**Comment by bovine3dom** [comment]
@GodSaveEarth that will set vmaps to the four character string `"{}"` which, honestly, i don't really know what it would do. There's not any particularly ergonomic way to totally wipe out a mode like you're suggesting.

Why do you want to remove all of the default binds? If you don't like visual mode you can disable it with `:set visualenterauto false`

**Comment by bovine3dom** [comment]
Great! Closing as I think the problem was down to a mangled user config :)

**Comment by GHolk** [comment]
@bovine3dom @GodSaveEarth @pzzxqj 

I figure out that if `tri.config.USERCONFIG.vmaps['🕷🕷INHERITS🕷🕷']` is set,
unbind will not make `tri.config.get('vmaps').s` become null.

For anyone want to fix this temporarily, try this command:
`js delete tri.config.USERCONFIG.vmaps['🕷🕷INHERITS🕷🕷']; tri.config.set('configversion', '2.0')`

The last set command can set anything.
It means to force tridactyl update the config cache.

**Comment by GHolk** [comment]
> I figure out that if `tri.config.USERCONFIG.vmaps['🕷🕷INHERITS🕷🕷']` is set, unbind will not make `tri.config.get('vmaps').s` become null.
> 
> For anyone want to fix this temporarily, try this command: `js delete tri.config.USERCONFIG.vmaps['🕷🕷INHERITS🕷🕷']; tri.config.set('configversion', '2.0')`

This happen because `src/lib/config.ts` calls `mergeDeepCull` to handle inherits, and mergeDeepCull will remove the null property (the unbinded one); then the default one get chose.

**Comment by bovine3dom** [comment]
Yeah. We were talking about this here right https://discord.com/channels/854326924402622474/854326925031374860/1418230643787169882

**Comment by GodSaveEarth** [comment]
@GHolk That command helped, thank you.

**Comment by bovine3dom** [comment]
Fixed in #5389
